src/main/java/com/project/model/Application.java

```diff
@@ -21,53 +22,53 @@
 
     @NotBlank(message = "Vardas negali būti tuščias")
     @Size(max = 255, message = "Vardas negali būti ilgesnis nei 255 simboliai")
-    @Column(name="full_name")
+    @Column(name = "full_name")
     private String name;
 
     @NotBlank(message = "Telefono numeris negali būti tuščias")
     @Size(max = 20, message = "Telefono numeris negali būti ilgesnis nei 20 simbolių")
-    @Column(name="phone_number")
+    @Column(name = "phone_number")
     private String phone_number;
 
     @NotBlank(message = "Mokymosi įstaiga negali būti tuščia")
     @Size(max = 255, message = "Mokymosi įstaigos pavadinimas negali būti ilgesnis nei 255 simboliai")
-    @Column(name="education")
+    @Column(name = "education")
     private String education;
 
     @NotBlank(message = "Pomėgių laukas negali būti tuščias")
     @Size(max = 1500, message = "Pomėgių lauke negali būti daugiau nei 1500 simbolių")
-    @Column(name="free_time")
+    @Column(name = "free_time")
     private String free_time;
 
     @NotNull(message = "Ar reikalinga trišalė sutartis privalo būti pasirinkta")
-    @Column(name="agreement")
+    @Column(name = "agreement")
     private boolean agreement;
 
     @Size(max = 1500, message = "Komentaras negali būti ilgesnis nei 1500 simboliai")
-    @Column(name="comment")
+    @Column(name = "comment")
     private String comment;
 
     @NotNull(message = "Ar tinka akademijos laikas privalo būti pasirinkta")
-    @Column(name="academy_time")
+    @Column(name = "academy_time")
     private boolean academy_time;
```

Comment:
Maybe `isAcademyTimeSuitable` would be a better name. Since this is a logic variable. It's hard to understand that does `academy_time` mean.

Reply:
changed several fields' names in Application class to better reflect what values are held in them